client/lib/analytics/ad-tracking.js

```diff
@@ -587,13 +587,9 @@ export function trackCustomAdWordsRemarketingEvent( properties ) {
 }
 
 function splitWpcomJetpackCartInfo( cart ) {
-	// Note: reduce() requires a non 0-length array.
-	const jetpackCost =
-		0 === cart.products.length
-			? 0
-			: cart.products
-					.map( product => ( productsValues.isJetpackPlan( product ) ? product.cost : 0 ) )
-					.reduce( ( accumulator, cost ) => accumulator + cost );
+	const jetpackCost = cart.products
```

Comment:
Why are these changes in the version bump PR?

Reply:
@vindl We had to rebase the branch to delete a commit introduced by SPT.

Reply:
I did `git rebase -i` to the commit right before the one introducing the SPT issue. And then deleted the SPT commit that is problematic. Then force pushed, which included those changes. Is there a better way to do it?